### DISCUSSION TITLE
Configuring Network/CPU throttling and Device Emulation while running lighthouse programatically

### DISCUSSION THREAD

**Discussion by aadisarma7** [GoogleChrome/lighthouse#15728] [Q&A] [answered]
I am using lighthouse user flows and am trying to pass specific network and CPU throttling settings. However, no matter what I pass to the lighthouse call, I still am getting metric values with negligible difference (FCP, LCP, etc). 
I am passing it to lighthouse as follows:
```
const networkConditions = {
    offline: false,
    latency: 800,
    downloadThroughput: Math.floor(1.5 * 1024 * 1024 / 8), // 1.6Mbps
    uploadThroughput: Math.floor(750 * 1024 / 8), // 750Kbps
    cpuSlowdownMultiplier: 10
  };
  const config = {
    extends: 'lighthouse:default',
    settings: {
      formFactor: 'desktop',
      screenEmulation: {
        disabled: true,
      },

    },
    throttling: networkConditions,
  };
  const browser = await puppeteer.launch({headless: false});
  const page = await browser.newPage();

  const flow = await startFlow(page, {config});
  await flow.navigate("https://www.vogue.com");
  ```
  Also, I was wondering if specific device emulation is supported or is it just low-end, mid-tier, and high-end mobile, and high-end, low-end desktop.

**Comment by adamraine** [comment, marked as answer]
`throttling` is set in the wrong spot. it needs to be under `config.settings`. When I do that I notice a huge jump in TBT likely from the large CPU multiplier you have set.
